data/mapcss-mapping.csv

```diff
@@ -513,12 +513,12 @@ healthcare|speech_therapist;438;
 deprecated|deprecated;439;x
 deprecated|deprecated;440;x
 aerialway|t-bar;441;
-deprecated|deprecated;442;x
-deprecated|deprecated;443;x
+natural|water|ditch;[natural=water][water=ditch];;name;int_name;442;
+natural|water|moat;[natural=water][water=moat];;name;int_name;443;
 highway|path|difficult_alpine_hiking;[highway=path][sac_scale=difficult_alpine_hiking];;name;int_name;444;
 earthquake:damage|spontaneous_camp;445;
-deprecated|deprecated;446;x
-deprecated:railway|yard:06.2023;447;x
+natural|water|drain;[natural=water][water=drain];;name;int_name;446;
+natural|water|wastewater;[natural=water][water=wastewater];;name;int_name;447;
```

Comment:
Hm, honestly saying putting a new type instead of deprecated 2 months ago is a bit scary :)
Please, keep 447 as-is, assign the new type with 448.